graphics/nine_patch.h

```diff
@@ -47,9 +47,13 @@
 #define GRAPHICS_NINE_PATCH_H
 
 #include "common/array.h"
+#include "common/rect.h"
 
 namespace Graphics {
 
+	class TransparentSurface;
```

Comment:
'namespace' gives no indent in our conventions, so this line and the following one should be corrected.